backend/config/webhooks.py

```
@@ -0,0 +1,33 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from .models import CustomerSubscription
+
+@csrf_exempt
+def stripe_webhook(request):
+    payload = request.body
+    sig_header = request.META.get('HTTP_STRIPE_SIGNATURE')
+    
+    try:
+        event = stripe.Webhook.construct_event(
+            payload, sig_header, settings.STRIPE_WEBHOOK_SECRET
```

Ruff (0.15.21)

[error] Undefined name `settings`

(F821)